server-common/src/main/java/com/datastrato/gravitino/server/authentication/OAuthConfig.java

```diff
@@ -5,14 +5,14 @@
 
 package com.datastrato.gravitino.server.authentication;
 
-import com.datastrato.gravitino.Configs;
 import com.datastrato.gravitino.config.ConfigBuilder;
 import com.datastrato.gravitino.config.ConfigConstants;
 import com.datastrato.gravitino.config.ConfigEntry;
 import io.jsonwebtoken.SignatureAlgorithm;
 import org.apache.commons.lang3.StringUtils;
 
-public interface OAuthConfig extends Configs {
+public interface OAuthConfig {
```

Comment:
Why do we remove the base class extension?

Reply:
You should see here:

![Uploading image.png…]()

Reply:
OK